src/app/components/TotalErrorCount.tsx

```diff
@@ -18,21 +18,18 @@ function TotalErrorCount(props) {
   return (
     <React.Fragment>
       {totalErrorCount > 0 ? (
-        <div className="total-error-count">
-          <h5 className="total-error-header">Total Errors:</h5>
+        <div className="totals-row">
+          <div className="section-title">Total Errors:</div>
```

Comment:
<img width="360" alt="Screen Shot 2020-02-09 at 2 44 46 PM" src="https://user-images.githubusercontent.com/7717963/74111624-ffd47c00-4b4a-11ea-82c2-4d0bb3ba94a4.png">
I like the new success state! But this new style of total errors does a good job really emphasizing the total error count to what we had previously.